tools/ts-schema-tool/src/SchemaCommands.ts

```diff
@@ -1,25 +1,28 @@
-import Ajv, {DefinedError, JSONSchemaType } from 'ajv';
+import Ajv, { DefinedError, JSONSchemaType } from 'ajv';
 import { RequiredMembers } from 'ajv/dist/types/json-schema';
 import * as fs from 'fs';
+
+import { AnyJson } from './JsonSchema';
 import { ObjectPrismSchemaField, PrismSchema, PrismSchemaField } from './PrismSchema';
 import { PrismSchemaError } from './PrismSchemaError';
 import { EmptySchemaResolver, SchemaResolver } from './SchemaResolver';
 
+const DEFAULT_CONTEXT_SCHEMA = "https://schema.org/Text";
```

Comment:
It;s should be schema for string types,
   (i.e. this should be array --  default-context-schemas, PrimitiveType => std(

Reply:
Also we should add test